source/material_model/reaction_model/tian2019_solubility.cc

```diff
@@ -65,7 +65,7 @@ namespace aspect
         // The fluid volume fraction in equilibrium with the solid (stored in the melt_fractions vector)
         // is equal to the sum of the porosity and the change in bound fluid content
         // (current bound fluid - updated average bound fluid).
-        melt_fractions = std::max(in.composition[q][bound_fluid_idx] + in.composition[q][porosity_idx] - average_eq_bound_water_content, 0.0);
+        melt_fractions = std::max(in.composition[q][bound_fluid_idx] + mass_frac_porosity - average_eq_bound_water_content, 0.0);
```

Comment:
The comment above still mentions porosity; I think that should be 
`is equal to the sum of the mass fraction of melt and the change in bound fluid content`